regional_mom6/regional_mom6.py

```diff
@@ -270,25 +274,30 @@ def motu_requests(
     productid="cmems_mod_glo_phy_my_0.083_P1D-m",
     buffer=0.3,
 ):
-    """Generates MOTU data request for each specified boundary, as
-    well as for the initial condition. By default pulls the GLORYS
-    reanalysis dataset.
+    """
+    Generate MOTU data request for each specified boundary, as well as for
+    the initial condition. By default pulls the GLORYS reanalysis dataset.
 
     Args:
-        xextent (List[float]): Extreme values of longitude coordinates for rectangular domain
-        yextent (List[float]): Extreme values of latitude coordinates for rectangular domain
-        daterange (Tuple[str]): Start and end dates of boundary forcing window. Format: ``%Y-%m-%d %H:%M:%S``
-        outfolder (str): Directory in which to receive the downloaded files
+        xextent (List[float]): Extreme values of longitude coordinates for
+            rectangular domain (in degrees).
+        yextent (List[float]): Extreme values of latitude coordinates for
+            rectangular domain (in degrees).
+        daterange (Tuple[str]): Start and end dates of boundary forcing window.
+            Format: ``%Y-%m-%d %H:%M:%S``.
+        outfolder (str): Directory the files are downloaded.
         usr (str): MOTU authentication username
         pwd (str): MOTU authentication password
         segs (List[str]): List of the cardinal directions for your boundary forcing
-        url (Optional[str]): MOTU server for the request (defaults to CMEMS)
-        serviceid (Optional[str]): Service containing the desired dataset
-        productid (Optional[str]): Data product within the chosen service.
-
-    Returns:
+        url (Optional[str]): MOTU server for the request. Defaults to the CMEMS,
+            i.e. ``"https://my.cmems-du.eu/motu-web/Motu"``.
+        serviceid (Optional[str]): Service containing the desired dataset. Default:
+            ``"GLOBAL_MULTIYEAR_PHY_001_030-TDS"``.
+        productid (Optional[str]): Data product within the chosen service. Default:
+            ``"cmems_mod_glo_phy_my_0.083_P1D-m"``.
+
+    Return:
         str: A bash script which will call ``motuclient`` to invoke the data requests.
-
     """
 
     if type(segs) == str:
```

Comment:
is this better way for doing this @angus-g ?

```suggestion
    if isistance(sets, str)
```

Reply:
Yeah, it is.